hashtables/ex1/ex1.py

```diff
@@ -1,17 +1,19 @@
 #  Hint:  You may not need all of these.  Remove the unused functions.
 from hashtables import (HashTable,
                         hash_table_insert,
-                        hash_table_remove,
-                        hash_table_retrieve,
-                        hash_table_resize)
+                        hash_table_retrieve)
 
 
 def get_indices_of_item_weights(weights, length, limit):
     ht = HashTable(16)
 
-    """
-    YOUR CODE HERE
-    """
```

Comment:
Nice.  You could also combine these into one for loop.  First check if j: return (j,i)   else:  insert(i)

Reply:
Thanks for the suggestion!